Apptentive/Apptentive/Apptentive.m

```diff
@@ -1074,16 +1097,6 @@ @implementation ApptentiveNavigationController
 
 // Container to allow customization of Apptentive UI using UIAppearance
 
-- (nullable instancetype)initWithCoder:(NSCoder *)aDecoder {
```

Comment:
I thought we are separating this method out in a separate PR.

Reply:
Unfortunately I had merged the branch into `develop` before seeing your comment. 

At this point, however, it would look the same whether or not we had merged it as a separate PR, since this is a roll-up of all of the changes since the last release. 

It *is* definitely something survey-related, which I found when testing the new stuff in Dark Mode. 

Agree this wasn't handled as it should be, but I'd maybe ask you to let it slide this one time :).

Reply:
No issues. Thanks a lot for your consideration.